tests/plugins/evaluation_formula_array.test.ts

```diff
@@ -509,10 +509,10 @@ describe("evaluate formulas that return an array", () => {
       setCellContent(model, "C1", "=MFILL(3,3, 42)");
       expect(getEvaluatedCell(model, "C1").value).toBe("#ERROR");
 
-      addColumns(model, "after", "D", 1);
+      addColumns(model, "after", "C", 1);
```

Comment:
what's the reason behind these changes in those tests?

Reply:
The allowDispatch of `ADD_COLUMNS_ROWS` was wrong (it used `>`instead of `>=`). I incidentally fixezd it in the PR, and those tests were adding columns out of the sheet (the sheet had 3 columns A,B,C and we tried to add a header after D)